packages/ng/style/components/cdk/_global.scss

```diff
@@ -13,5 +13,11 @@
 }
 
 .cdk-global-scrollblock {
-	overflow: hidden;
+	&,
+    & body {
```

Comment:
We don't usually put the second `&` for the children selectors.

Reply:
Fixed here https://github.com/LuccaSA/lucca-front/pull/1533